pkgs/tools/misc/atuin/default.nix

```diff
@@ -21,7 +21,7 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-P8PTEbTWI+vwWiPvjr4yTwO/JfPzfpUPO0ATi1ILhfk=";
   };
 
-  cargoHash = "sha256-T3Y6WiU6UdLmZiXYekL5cIonqFisU94PpiVlB1sNr9U=";
+  cargoHash = if stdenv.isLinux then "sha256-T3Y6WiU6UdLmZiXYekL5cIonqFisU94PpiVlB1sNr9U=" else "sha256-a81gKajbifhDqlKlpnA4FzVX+NktqOCRqlajuuYopCg=";
```

Comment:
Did someone try to find the root cause of this?

Reply:
I briefly looked into the cargo lock and found a deoendency called linux raw , i guessed it was probably that one, but didnt verify

Reply:
oh, well but on the next bump this will likely need to extra attention because I personally do not care to much about darwin.

Reply:
I use this on my darwin machine, happy to make a fix PR if that breaks